tests/latent_heat_viscosity.prm

```diff
@@ -17,11 +17,11 @@ subsection Material model
     set Viscosity                     = 1e21
     set Thermal viscosity exponent    = 15.0
     set Reference temperature         = 1600
-    set Phase transition depths       = 410000, 660000
-    set Phase transition widths       = 20000, 20000
-    set Phase transition temperatures = 1600, 1600
-    set Phase transition Clapeyron slopes = 3e6, -2e6
-    set Phase transition density jumps = 100, 200
+    set Phase transition depths           = all:410000|660000
+    set Phase transition widths           = all:20000|20000
+    set Phase transition temperatures     = all:1600|1600
+    set Phase transition Clapeyron slopes = all:3e6|-2e6
```

Comment:
Ah, so this is not backwards compatible. I am fine with that, but you should add a changelog entry. 
This is something we can not address in the update script, right? Is there anything we can do about this?

Reply:
And have you tested that all cookbooks still work?